### PR TITLE
Mark Safari 15 as current version

### PR DESCRIPTION
**Pull request by lukewarlow**
#### Summary

iOS 15 has been released which includes Safari 15, and Safari 15 on mac has been release. This MR sets 15 as the current Safari (and Safari iOS) version.

#### Test results and supporting details

image

### CODE REVIEW

**Comment by lukewarlow** [comment]
The release notes unfortunately haven't been updated yet, and still refer to it as beta.

File: browsers/safari.json [hunk 1]
```diff
@@ -172,14 +172,16 @@
...
           "engine_version": "611.1.21.161.3"
         },
         "15": {
+          "release_date": "2021-09-20",
           "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-15-beta-release-notes",
-          "status": "beta",
-          "engine": "WebKit"
+          "status": "current",
+          "engine": "WebKit",
+          "engine_version": "612.1.29.41.4"
```
Comment (queengooborg) [inline browsers/safari.json:184]:
```suggestion
          "engine_version": "612.1.29"
```

Comment (queengooborg) [inline browsers/safari.json:184]:
```suggestion
          "engine_version": "612.1.29"
```

File: browsers/safari_ios.json [hunk 2]
```diff
@@ -143,14 +143,16 @@
...
           "engine_version": "611.1.21.161.6"
         },
         "15": {
+          "release_date": "2021-09-20",
           "release_notes": "https://developer.apple.com/documentation/safari-release-notes/safari-15-beta-release-notes",
-          "status": "beta",
-          "engine": "WebKit"
+          "status": "current",
+          "engine": "WebKit",
+          "engine_version": "612.1.29.41.4"
```
Comment (queengooborg) [inline browsers/safari_ios.json:155]:
```suggestion
          "engine_version": "612.1.29"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Thanks for taking care of this!  Can we simplify the engine version to just the major, minor, and patch versions?  (I know Safari 14.1 has the micro and nano versions, but this is actually causing problems with our mirroring script.)

**Review by queengooborg — APPROVED** [review]

**Pull request merged**